.github/workflows/publish.yml

```diff
@@ -49,3 +50,8 @@ jobs:
           corepack yarn npm publish
         env:
           YARN_NPM_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+
+      - name: Add Corepack executable to GitHub release
+        run: gh release upload ${{ needs.release-please.outputs.release_tag }} dist/
```

Comment:
Shouldn't it upload the result of `yarn pack` since Corepack depends on the `package.json` to locate itself?
https://github.com/nodejs/corepack/blob/15ceb832a34a223efbe3d3f9cb792d9101a7022a/sources/commands/Enable.ts#L53-L55

Reply:
Good catch, addressing that in https://github.com/nodejs/corepack/pull/267